src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReadyToRunHelperNode.cs

```diff
@@ -36,10 +36,10 @@ public enum ReadyToRunHelperId
         DefaultConstructor,
     }
 
-    public partial class ReadyToRunHelperNode : AssemblyStubNode, INodeWithDebugInfo
+    public partial class ReadyToRunHelperNode : AssemblyStubNode
```

Comment:
Doesn't the _runtime_ generate these helpers on the CoreCLR side? I thought they're not part of the image because it would defeat the purpose. At least in the CoreRT version of the R2R helpers, this is where we hardcode the details like vtable offsets and EEType addresses. I'm not sure we'll be able to share any of this.